js/apps/account-ui/test/account-security.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "@playwright/test";
+import { login } from "./login";
+
+test("Check page heading", async ({ page }) => {
+  await login(page, "jdoe", "jdoe", "photoz");
+  await page
+    .getByRole("button", {
+      name: "Account security",
+    })
+    .click();
+
+  await page.getByTestId("account-security/linked-accounts").click();
+
+  // Check the page heading
+  const pageHeadingElement = await page.getByTestId("page-heading");
+  await expect(pageHeadingElement).toHaveText("Linked accounts");
```

Comment:
If I am reading the docs correctly you can pass the Promise for an element into the `expect()` call. So this could be shortened to:

```suggestion
  await expect(page.getByTestId("page-heading")).toHaveText("Linked accounts");
```

This doesn't impact the functionality of this code of course, but it's nice to know.